specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/examples/NspAssociationPut.json

```diff
@@ -0,0 +1,57 @@
+{
+  "title": "Create/Update NSP association",
+  "parameters": {
+    "api-version": "2021-02-01-preview",
+    "subscriptionId": "subId",
+    "resourceGroupName": "rg1",
+    "networkSecurityPerimeterName": "nsp1",
+    "associationName": "association1",
+    "parameters": {
+      "properties": {
+        "privateLinkResource": {
+          "id": "/subscriptions/{paasSubscriptionId}/resourceGroups/{paasResourceGroupName}/providers/{providerName}/{resourceType}/{resourceName}"
+        },
+        "profile": {
+          "id": "/subscriptions/subId/resourceGroups/rg1/providers/Microsoft.Network/networkSecurityPerimeters/nsp1/profiles/{profileName}"
```

Comment:
Do you expect to do any linked access chectks etc when creating these resources ?  :

Reply:
[https://armwiki.azurewebsites.net/authorization/RBACLinkedAccessCheck.html](https://armwiki.azurewebsites.net/authorization/RBACLinkedAccessCheck.html)

Reply:
?